pkg/apis/triggers/v1alpha1/cluster_interceptor_defaults.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"context"
+)
+
+var defaultPort = int32(80)
```

Comment:
Just a thought that 80 and 443 are kind of standard port 
So can we use some other port?

Reply:
users can always specify a different port. I think 80 makes a good default since otherwise every interceptor author will have to specify the "custom" default port that we added. 

(When we add TLS support, we could try 443 as default perhaps -- that's what the WebhookConfiguration CRDs do.)